small UI/UX improvements

Unpublished opened the pull request.

M66B commented on app/src/main/java/eu/faircode/email/HtmlHelper.java:
```diff
@@ -324,7 +324,10 @@ static Drawable decodeImage(String source, Context context, long id, boolean sho
+        if (text == null || text.length() <= PREVIEW_SIZE) {
+            return text;
+        }
+        return text.substring(0, PREVIEW_SIZE) + '\u2026';
```
There is no need for this, since the layout is set to ellipsize already:

https://github.com/M66B/open-source-email/blob/master/app/src/main/res/layout/item_message_normal.xml#L250

Unpublished replied:
This is intended for the preview in expanded notifications where it is just cut off after `PREVIEW_SIZE` characters and it doesn't hurt the preview in TextView you linked.

M66B replied:
I have solved it like this because I don't think what goes into the database should be changed: https://github.com/M66B/open-source-email/commit/1ed0fe7fce07b25f6bf31234d7ca0cd46c59e19a

M66B commented on app/src/main/java/eu/faircode/email/FragmentMessages.java:
```diff
@@ -759,6 +759,13 @@ public void onChildDraw(
+        public void onSelectedChanged(@Nullable RecyclerView.ViewHolder viewHolder, int actionState) {
+            super.onSelectedChanged(viewHolder, actionState);
+            final boolean swiping = actionState == ItemTouchHelper.ACTION_STATE_SWIPE;
+            swipeRefresh.setEnabled(!swiping);
```
This is a nice addition. Did you test if it works properly?

M66B replied:
Tested and working properly, but should be disabled when pull to refresh was disabled (see commit).

Unpublished replied:
Good catch, thanks!
And yes it works well for me too.

M66B commented on app/src/main/java/eu/faircode/email/ServiceSynchronize.java:
```diff
@@ -687,7 +687,7 @@ public void run() {
                         StringBuilder sb = new StringBuilder();
                         if (!TextUtils.isEmpty(message.subject))
-                            sb.append(message.subject).append("<br>");
+                            sb.append("<b>").append(message.subject).append("</b>").append("<br>");
```
The formatting of the subject was deliberately kept the same expanded and not expanded, so there is a smooth transition when expanding/collapsing.

Unpublished replied:
Hmm...I thinks it is difficult to distinguish between subject and preview without highlighting it.

M66B replied:
How about italic as something in between? The notification title is already bold and the difference between expanded and collapse will be smaller.

M66B replied:
I think it is a better idea to show the text italic, just like in the message view.

M66B commented:
Thanks for trying to contribute to the FairEmail project!

Unpublished commented:
You're welcome. Will rebase if you are in favor of the other changes.

M66B commented:
Please check my changes and if you are satisfied close this pull request.

You can find my email address in the source code, so if you like you can contact me directly.

Let me hear when you have other ideas for improvements.

Unpublished commented:
Your changes look good, thanks for your work!

Fixed some nits (for empty preview case and removed space before … like it is done in the message view). Fixes are here in case you want to pick them the message: https://github.com/Unpublished/open-source-email/commit/66a01ceb5a0080777880d0d47bf83c62dd5f38dd

Sure, will get in touch when I come up with something.

M66B commented:
I've update the commit: https://github.com/M66B/open-source-email/commit/4653938259e34f21e1110bff57a698f22932ff82

Linguistically seen there should be a space, but who'll notice?
